README.md

```diff
@@ -38,16 +38,17 @@ that use Pravega as the stream storage and message bus, and Apache Spark for com
 The [master](https://github.com/pravega/spark-connectors) branch will always have the most recent
 supported versions of Spark and Pravega.
 
-| Spark Version | Pravega Version | Git Branch                                                                        |
-|---------------|-----------------|-----------------------------------------------------------------------------------|
-| 3.0           | 0.8             | [master](https://github.com/pravega/spark-connectors)                             |
-| 2.4           | 0.8             | [r0.8-spark2.4](https://github.com/pravega/spark-connectors/tree/r0.8-spark2.4)   |
+| Spark Version | Pravega Version | Java Version | Git Branch                                                                        |
+|---------------|-----------------|--------------|-----------------------------------------------------------------------------------|
+| 3.0           | 0.9             | Java 11      | [master](https://github.com/pravega/spark-connectors)                             |
```

Comment:
We should still keep the usability under Java 8 for our Spark 3 connector, it's just the project build and testing requires Java 11. We need to mention this here.

Reply:
That is good idea. Let's have these columns instead.
Java Version to Build Connector: Possible values are Java 8, Java 11
Java Version to Use Connector: Possible values are Java 8, Java 11, Java 8 or 11

Reply:
Please check again